.github/CODEOWNERS

```diff
@@ -12,3 +12,4 @@
 /src/datasources/test-plans @priya-ni @hemkumarrao-ni @kartheeswaran-ni @ushakanagaraj-ni
 /src/datasources/alarms @hemkumarrao-ni @kartheeswaran-ni @ushakanagaraj-ni
 /src/datasources/data-frame @hemkumarrao-ni @kartheeswaran-ni @ushakanagaraj-ni @cameronwaterman
+/src/datasource/system @chotea
```

Comment:
@BlankaOnodi you and @stefanc18 were added to the repo with git write access.
Moving the PR back to draft as the current changes don't reflect what was intended to be merged from offline discussions.